examples/retrieval.yaml

```diff
@@ -45,4 +45,4 @@ defaults:
 
 standard-dropout0.1:
   train:
```

Comment:
You can just remove the whole `train:` block and rename `standard-dropout0.1` to `retrieval`.